orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/cluster/AbstractClusterWideClouddriverTask.groovy

```diff
@@ -102,6 +102,7 @@ abstract class AbstractClusterWideClouddriverTask extends AbstractCloudProviderA
     }
 
     List<Map> serverGroups = cluster.get().serverGroups
+    log.debug("Server groups fetched from cluster: ${serverGroups}")
```

Comment:
Since you're trying to troubleshoot a potential filtering problem, why not just pull out the names of the server groups here (and below) instead of serializing their full representations?